src/components/Router/StepComponentMap.ts

```diff
@@ -141,12 +141,22 @@ const buildFaceComponents = (
   const shouldSelfieScreenUseCamera =
     !shouldDisplayUploader && deviceHasCameraSupport
 
+  const videoCameraSupport = window.MediaRecorder != null
+
+  const photoCaptureFallback =
+    faceStep?.options?.photoCaptureFallback == null
+      ? true
+      : faceStep?.options?.photoCaptureFallback
```

Comment:
This can be simplified to: 

```suggestion
  const photoCaptureFallback = faceStep?.options?.photoCaptureFallback  !== false
```

Reply:
updated! :)